daemon/main.cpp

```diff
@@ -216,7 +216,7 @@ main(int argc, char* argv[])
   {
     llarp::ensureConfig(
         llarp::GetDefaultDataDir(), llarp::GetDefaultConfigPath(), overwrite, opts.isRelay);
-    conffname = llarp::GetDefaultConfigPath().c_str();
+    conffname = llarp::GetDefaultConfigPath().string().c_str();
```

Comment:
this string will fall out of scope

Reply:
`conffname` is a `std::string` so this is better as just `.string();` anyway (i.e. just drop the `.c_str()`).